CHANGELOG.md

```diff
@@ -9,8 +9,9 @@
 
 ### Minor Changes
 
-- Fixed a bug in `NftNamesapce.getNftsForOwner()` where the `openSea` metadata was not included in the returned NFT.
-- Marked the `NftFilters` type as deprecated. Please use `NftExcludeFilters` or `NftIncludeFilters` instead.
+- Fixed a bug in `NftNamespace.getNftsForOwner()` where the `openSea` metadata was not included in the returned NFT.
+- Marked the `NftExcludeFilters` type as deprecated. Please use `NftFilters` instead.
```

Comment:
FYI I think these were flipped in the existing entry!

Reply:
oops, thanks for catching!